buildSrc/src/main/kotlin/com.hedera.hashgraph.conventions.gradle.kts

```diff
@@ -75,19 +72,19 @@ repositories {
     maven {
         url = uri("https://oss.sonatype.org/content/repositories/comhederahashgraph-1531")
     }
-    mavenLocal()
+    mavenLocal() // TODO should be removed. Why is this needed?
```

Comment:
❓ mavenLocal() is typically used if there needs to be some interaction with another **Maven** build. Usually, you do not need it in a pure Gradle setup. Is there a particular reason why it is here?

Reply:
I believe this was added to support the JMH tests but I agree it should be removed if possible.